charts/zenko-filer/values.yaml

```diff
@@ -0,0 +1,46 @@
+# Default values for zenko-filer.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+replicaCount: 1
+
+nfsd:
+  name: "zenko-nfsd"
+  image:
+    repository: zenko/zenko-nfsd
+    tag: edge
+    pullPolicy: IfNotPresent
+  loglevel: INFO
```

Comment:
shall we use capital letters? I think all other charts use lowercase, and a structure like

```
logging:
    level: info
```

Reply:
It would probably require a change in the nfsd code, ```AFAIK``` nfs ganesha log variables are in all-caps. We can still make the division of ```logging``` and ```level```.